roles/openshift_obs/tasks/main.yml

```diff
@@ -0,0 +1,61 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Install cluster observability operator.
+  kubernetes.core.k8s:
+    definition:
+      apiVersion: operators.coreos.com/v1alpha1
+      kind: Subscription
+      metadata:
+        name: observability-operator
+        namespace: openshift-operators
+        labels:
+          operators.coreos.com/observability-operator.openshift-operators: ""
+      spec:
+        channel: development
+        installPlanApproval: Automatic
+        name: cluster-observability-operator
+        source: redhat-operators
+        sourceNamespace: openshift-marketplace
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    state: present
+    wait: true
```

Comment:
This wait here does nothing, as the k8s module only supports the waiting argument, without explicit conditions, for `Deployment`s, `Pod`s and `DaemonSet`.
https://docs.ansible.com/ansible/latest/collections/kubernetes/core/k8s_module.html#parameter-wait

This is how we did it for other role, maybe there is a better way, but we haven't been able to do so using the module.
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/ci_nmstate/tasks/nmstate_k8s_install.yml#L34-L51

PS: As you are already waiting for the Pod and the Deployment this wait here being skipped is not an issue.

Reply:
Done. thank you.